buildagent-generation.yml

```diff
@@ -14,22 +15,42 @@ parameters:
   displayName: Agent Pool
   type: string
   default: CI Windows2019
-
-trigger: none
-
-variables:
-  - group: ${{ parameters.variable_group }}
+- name: depends_on
+  displayName: First Job Depends On
+  type: object
+  default: 
+- name: repository_base_path
+  displayName: Scripts Path
+  type: string
+  default: .
+- name: resource_repo_name
+  displayName: Resource Repository Name
+  type: string
+  default: 
```

Comment:
Can you add a default value here? Default standalone pipeline requires something to be here, to start. I don't want people to fill things every time they start the pipeline (when it is not used anyways). I checked conditional parameters but that doesn't seem to exist.

Reply:
Do you have an error when trying to run the pipeline without this variable? I thought "default: " would instruct Azure DevOps to just let it empty by default. Maybe I should just put "default: ''" instead. I'll try that.

Reply:
It just can't run, as it puts the variable as required. It NEEDS a value, unfortunately.